File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/RunStepResponse.kt
package com.sdkwork.ai

data class RunStepResponse(
    val id: String? = null,
    val object: String? = null,
    val type: String? = null,
    val status: String? = null,
    val metadata: Map<String, String>? = null,
    val createdAt: Int? = null,
    val assistantId: String? = null,
    val threadId: String? = null,
    val runId: String? = null,
    val stepDetails: Any? = null,
    val lastError: Any? = null,
    val expiredAt: Int? = null,
    val cancelledAt: Int? = null,
    val failedAt: Int? = null,
    val completedAt: Int? = null
)

File: sdkwork-ai-sdk-csharp/Models/RunStepResponse.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class RunStepResponse
    {
        public string? Id { get; set; }
        public string? Object { get; set; }
        public string? Type { get; set; }
        public string? Status { get; set; }
        public Dictionary<string, string>? Metadata { get; set; }
        public int? CreatedAt { get; set; }
        public string? AssistantId { get; set; }
        public string? ThreadId { get; set; }
        public string? RunId { get; set; }
        public object? StepDetails { get; set; }
        public object? LastError { get; set; }
        public int? ExpiredAt { get; set; }
        public int? CancelledAt { get; set; }
        public int? FailedAt { get; set; }
        public int? CompletedAt { get; set; }
    }
}
